tests_aws/test_registry/test_data_sync.py

```diff
@@ -0,0 +1,165 @@
+import pytest
+import uuid
+
+from dsgrid.cloud.s3_storage_interface import S3StorageInterface
+from dsgrid.filesystem.local_filesystem import LocalFilesystem
+from dsgrid.registry.registry_manager import RegistryManager
+from dsgrid.common import REMOTE_REGISTRY, LOCAL_REGISTRY
+from dsgrid.tests.common import TEST_PROJECT_PATH, TEST_REMOTE_REGISTRY, AWS_PROFILE_NAME
+from tests_aws.test_registry.common import clean_remote_registry
+
+local_registry = TEST_PROJECT_PATH / "test_registry_data_sync" / "registry"
+local_registry_data_sync = TEST_PROJECT_PATH / "test_registry_data_sync" / "data_sync_registry"
+remote_registry = TEST_REMOTE_REGISTRY
+
+s3_cloudinterface = S3StorageInterface(
+    remote_path=TEST_REMOTE_REGISTRY,
+    local_path=local_registry,
+    user="Test",
+    uuid=str(uuid.uuid4()),
+    profile=AWS_PROFILE_NAME,
+)
+
+
+def test_data_sync_project_id(s3_cloudinterface, local_registry_data_sync, remote_registry):
```

Comment:
This test fails currently with the following error message.

```
_____________________________________________________________ ERROR at setup of test_data_sync_project_id _____________________________________________________________
file /Users/mmooney/Documents/github/github.com/dsgrid/dsgrid/tests_aws/test_registry/test_data_sync.py, line 24
  def test_data_sync_project_id(s3_cloudinterface, local_registry_data_sync, remote_registry):
E       fixture 's3_cloudinterface' not found
>       available fixtures: anyio_backend, anyio_backend_name, anyio_backend_options, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.
```

@daniel-thom any thoughts here? All other tests with the same `s3_cloudinterface` logic work fine and do not result in an error.